can_ids.csv

```diff
@@ -0,0 +1,57 @@
+8,MDE22,170,STATE,100,1,0.014,71,Module state report
+9,MDE22,170,STEERINGBAT_MEASUREMENTS,201,10,0.206,103,Auxiliar Battery Voltage
+10,MVC19_1,210,STATE,101,1,0.014,71,Module state report
+11,MVC19_2,211,STATE,102,1,0.014,71,Module state report
+12,MCC19_1,225,STATE,103,1,0.016,79,Module state report
+13,MCC19_1,225,MEASUREMENTS,202,30,0.666,111,All measurements from the converter
+14,MCC19_2,226,STATE,104,1,0.016,79,Module state report
+15,MCC19_2,226,MEASUREMENTS,203,30,0.666,111,All measurements from the converter
+16,MCC19_3,227,STATE,105,1,0.016,79,Module state report
+17,MCC19_3,227,MEASUREMENTS,204,30,0.666,111,All measurements from the converter
+18,MCC19_4,228,STATE,106,1,0.016,79,Module state report
+19,MCC19_4,228,MEASUREMENTS,205,30,0.666,111,All measurements from the converter
+20,MCC19_5,229,STATE,107,1,0.016,79,Module state report
+21,MCC19_5,229,MEASUREMENTS,206,30,0.666,111,All measurements from the converter
+22,MCC19_6,239,STATE,108,1,0.016,79,Module state report
+23,MCC19_6,239,MEASUREMENTS,207,30,0.666,111,All measurements from the converter
+24,MCB19_1,220,STATE,109,1,0.016,79,Module state report
+25,MCB19_1,220,MEASUREMENTS,208,10,0.222,111,All measurements from the converter
+26,MCB19_2,221,STATE,110,1,0.016,79,Module state report
+27,MCB19_2,221,MEASUREMENTS,209,10,0.222,111,All measurements from the converter
+28,MAC22,180,STATE,35,5,0.071,71,Module state report
+29,MAC22,180,CONTACTOR,34,50,0.63,63,Contactor task response
+30,MAM19,190,STATE,99,5,0.071,71,Module state report
+31,MAM19,190,MOTOR,98,50,0.71,71,Motor controller parameters
+32,MAM19,190,CONTACTOR,36,5,0.063,63,Contactor requests
+33,MAB19,230,STATE,111,0,0.0,71,Module state report
+34,MAB19,230,PUMPS,210,0,0.0,63,Pumps state
+35,MSC19_1,250,STATE,112,1,0.014,71,Module state report
+36,MSC19_1,250,ADC,211,10,0.206,103,Voltage measurements
+37,MSC19_2,251,STATE,113,1,0.014,71,Module state report
+38,MSC19_2,251,ADC,212,10,0.206,103,Voltage measurements
+39,MSC19_3,252,STATE,114,1,0.014,71,Module state report
+40,MSC19_3,252,ADC,213,10,0.206,103,Voltage measurements
+41,MSC19_4,253,STATE,115,1,0.014,71,Module state report
+42,MSC19_4,253,ADC,214,10,0.206,103,Current measurements
+43,MSC19_5,254,STATE,116,1,0.014,71,Module state report
+44,MSC19_5,254,ADC,215,10,0.206,103,Current measurements
+45,MCS19,200,STATE,117,1,0.014,71,Module state report
+46,MCS19,200,START_STAGES,37,50,0.71,71,Boat charging // Boat on
+47,MCS19,200,BAT,216,10,0.206,103,battery voltage values
+48,MCS19,200,CAP,217,10,0.206,103,capacitor bank voltage values
+49,MT19,255,STATE,218,0,0.0,71,Module state report
+50,MT19,255,RPM,219,0,0.0,71,RPM motor values
+51,MSWI19,241,STATE,9,5,0.071,71,Module state report
+52,MSWI19,241,MOTOR,10,50,0.79,79,Motor controller parameters
+53,MSWI19,241,PUMPS,220,0,0.0,63,Pumps controller parameters
+54,MSWI19,241,MPPTS,221,0,0.0,71,Mppts controller parameters
+55,MSWI19,241,MCS,40,1,0.013,63,MCS controller parameters
```

Comment:
If this file is generated, do we want it to be added in the repository?

Reply:
I don't know the utility, I only added to the repository because all the others generated files are in the repo(.json and .h)